allan_variance.py

```diff
@@ -120,3 +129,23 @@ def params_from_avar(tau, avar):
     prediction = A.dot(x) * avar
 
     return np.sqrt(x), prediction
+
+
+def allan_deviation(avar):
```

Comment:
Maybe. But this is a definition in the problem-oriented code. Like the law ``F := dp / dt``. Simple expression of three values, but it's fundamental for physics. I think it should be.